src/app/Fake.netcore/Fake.netcore.fsproj

```diff
@@ -1,34 +1,35 @@
-<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
-    <DefineConstants>$(DefineConstants);CORE_CLR;DOTNETCORE;EXPLICIT_DEPENDENCIES;NETSTANDARD;NETSTANDARD1_6</DefineConstants>
-    <DelaySign>true</DelaySign>
-    <DebugType>Portable</DebugType>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <AssemblyName>fake</AssemblyName>
-    <OutputType>Exe</OutputType>
-    <RuntimeIdentifiers>win7-x86;win7-x64;osx.10.11-x64;ubuntu.14.04-x64;ubuntu.16.04-x64</RuntimeIdentifiers>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
-    <DefineConstants>$(DefineConstants);RELEASE</DefineConstants>
-  </PropertyGroup>
-  <ItemGroup>
-    <Compile Include="AssemblyInfo.fs" />
-    <Compile Include="Cli.fs" />
-    <Compile Include="Program.fs" />
-    <!-- Note: This resource should be moved to Paket.Core... -->
-    <EmbeddedResource Include="..\..\..\.paket\Paket.Restore.targets" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Fake.Runtime\Fake.Runtime.fsproj">
-      <FromP2P>true</FromP2P>
-    </ProjectReference>
-    <ProjectReference Include="..\Fake.Core.CommandLineParsing\Fake.Core.CommandLineParsing.fsproj">
-      <FromP2P>true</FromP2P>
-    </ProjectReference>
-    <ProjectReference Include="..\Fake.Core.Context\Fake.Core.Context.fsproj">
-      <FromP2P>true</FromP2P>
-    </ProjectReference>
-  </ItemGroup>
-  <Import Project="..\..\..\.paket\Paket.Restore.targets" />
-</Project>
\ No newline at end of file
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" 
+    xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
any idea why every line changed? Line ending changed?

Reply:
Auto-Format on save in vscode. Probably uses the 4 spaces from the editorconfig.

Reply:
Ah I didn't noticed, thanks. Can we change that for project files to two? Or just leave it as is. Now that I understand whats going on :)
But I think two is what the files use when auto generated (for example via `dotnet new`), correct?